web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js

```diff
@@ -48,7 +48,12 @@ context('Create Configure and Delete PluginTemplate', () => {
     cy.visit('/');
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
-    cy.contains('Advanced').should('be.visible').click();
+    cy.contains('Advanced')
+      .should('be.visible')
+      .click()
+      .then(() => {
+        cy.contains('Advanced').trigger('mouseover');
```

Comment:
Are these sections required?

Reply:
The drop-down filter needs to be triggered here, how to trigger it to make it effective and better? Do you have a better suggestion?
Another question, I see that some places trigger two clicks, like this file: pluginTemplate-create-plugin-template-with-route.cy.js
cy.contains('Next').click();
cy.contains('Next').click();
I don't know what it means and how does it work?
Someone else answered a similar question, but didn't tell the reason
![image](https://user-images.githubusercontent.com/17808807/197185443-1e8dcca7-1f92-4bb0-ac12-eedd637960f3.png)

Reply:
please re-run it
![image](https://user-images.githubusercontent.com/17808807/197185862-d67af4d6-5373-45e1-b2b6-910882e1974a.png)
![image](https://user-images.githubusercontent.com/17808807/197185896-a65449b2-0020-417d-8672-b819e73baf41.png)